zk/stages/stage_sequence_execute_utils.go

```diff
@@ -498,3 +499,69 @@ func (bdc *BlockDataChecker) AddTransactionData(txL2Data []byte) bool {
 
 	return false
 }
+
+type ResequenceBatchJob struct {
+	batchToProcess  []*dsTypes.FullL2Block
+	StartBlockIndex int
+	StartTxIndex    int
+	txIndexMap      map[common.Hash][]int
+}
```

Comment:
WDYT about such struct (instead of having slice with two ints)?

```suggestion
	txIndexMap      map[common.Hash]*txMetadata
}

type txMetadata struct{
	blockNum int
	txIndex int
}
```

Reply:
Good suggestion. Will change accordingly.